core/src/main/java/com/linecorp/armeria/client/AbstractClientOptionsBuilder.java

```diff
@@ -257,6 +266,14 @@ public AbstractClientOptionsBuilder rpcDecorator(DecoratingRpcClientFunction dec
         return this;
     }
 
+    /**
+     * Clears the RPC-level decorators set so far.
+     */
+    public AbstractClientOptionsBuilder clearRpcDecorator() {
+        decoration.clearRpc();
+        return this;
+    }
```

Comment:
- `clearRpcDecorator()` -> `clearRpcDecorators()`
- Should we make `clearDecorators()` clear both RPC and non-RPC decorators? :thinking: